dkron/storage.go

```diff
@@ -16,7 +16,7 @@ type Storage interface {
 	SetExecutionDone(execution *Execution) (bool, error)
 	GetJobs(options *JobOptions) ([]*Job, error)
 	GetJob(name string, options *JobOptions) (*Job, error)
-	GetExecutions(jobName string, timezone *time.Location) ([]*Execution, error)
+	GetExecutions(jobName string, timezone *time.Location, opts *ExecutionOptions) ([]*Execution, error)
```

Comment:
It would be great to refactor `timezone` into `ExecutionOptions`

Reply:
Done